code/__DEFINES/living_flags.dm

```diff
@@ -0,0 +1 @@
+#define LIVING_TK_USER (1<<0)
```

Comment:
Not really happy with define organization since there's already the mobs one but not sure how to feel about it.

Reply:
Up to you. I'd rather keep the defines each on their own file, but can adapt at will.